package.json

```diff
@@ -73,8 +73,8 @@
     "compile-typedoc-platform": "rm -rf apps/docs/src/assets/typedoc/platform && typedoc --out apps/docs/src/assets/typedoc/platform libs/platform/src/lib --tsconfig libs/platform/src/lib/tsconfig.lib.json --hideGenerator --theme apps/docs/src/fd-typedoc",
     "compile-typedoc-json": "typedoc --json apps/docs/src/assets/typedoc/typedoc.json --hideGenerator --theme apps/docs/src/fd-typedoc libs/core/src/lib",
     "build:schematics": "./node_modules/.bin/tsc -p libs/core/tsconfig.schematics.json && yarn run copy:schemas && yarn run copy:collection",
-    "copy:schemas": "cp libs/core/schematics/ng-add/schema.json dist/libs/core/schematics/ng-add/schema.json && yarn run copy:core-migration-shema",
-    "copy:core-migration-shema": "cp libs/core/schematics/migrations/migrate-theme-fonts/schema.json dist/libs/core/schematics/migrations/migrate-theme-fonts/schema.json",
+    "copy:schemas": "cp libs/core/schematics/ng-add/schema.json dist/libs/core/schematics/ng-add/schema.json && yarn run copy:core-migration-schema",
+    "copy:core-migration-schema": "cp libs/core/schematics/migrations.json dist/libs/core/schematics/migrations.json",
```

Comment:
Thanks. without bash I will not be able to use these scripts on my machine even more(Windows...)

Reply:
actually I didn't change anything, just file names

Reply:
in case we have issues with commands we could fix it in separate pr.